### PR TITLE
Use external SD-JWT library to generate examples

### LINKED ISSUE #259: Remove generated examples from repository and generate examples in build pipeline

### PR DESCRIPTION
**Pull request by danielfett**
Fixes #259

### CODE REVIEW

**Comment by danielfett** [comment]
Since this is a 122 files-changed commit, here's the gist:

- I removed the sd-jwt python code
- I added code to the github action to pull the danielfett/sd-jwt library and generate the examples from the example specifications
- Those now live in examples/*/specification.yml
- I removed all the generated artifacts and updated the .gitignore accordingly.

**Comment by bc-pi** [comment]
Thanks for doing the work here and for the gist of the changes. It is, as I'm sure you know, not easy to review or approve a change like this. I wanted to acknowledge the work here but hoping we can discuss in our (usually) weekly meeting. 

In the meantime, it does look like the decoy_digests.md stuff didn't make it through this change. There are no digest values after the "The Issuer added the following decoy digests:" text at https://drafts.oauth.net/oauth-selective-disclosure-jwt/danielfett/generated-examples/draft-ietf-oauth-selective-disclosure-jwt.html#appendix-A.1-26 which I just happened to notice looking at this diff https://author-tools.ietf.org/api/iddiff?url_1=https://oauth-wg.github.io/oauth-selective-disclosure-jwt/draft-ietf-oauth-selective-disclosure-jwt.txt&url_2=https://oauth-wg.github.io/oauth-selective-disclosure-jwt/danielfett/generated-examples/draft-ietf-oauth-selective-disclosure-jwt.txt which I just happened to notice at https://drafts.oauth.net/oauth-selective-disclosure-jwt/

**Comment by danielfett** [comment]
Thanks for catching that. The change was in the SD-JWT library repository, but there was a bug leading to an empty decoy_digests.md file. Should be fixed now.

File: README.md [hunk 1]
```diff
@@ -17,63 +17,30 @@ Contributions can be made by creating pull requests.
...
-sd_jwt sd_jwt/examples/simple_structured.yml
-sd_jwt sd_jwt/examples/complex.yml
-```
+To build formatted text and HTML versions of the draft, run the following steps:
 
-You can create your custom setting file creating a folder with a copy of
-[sd_jwt/demo_settings.py](sd_jwt/demo_settings.py) renamed to `settings.py`
-and a `__init__.py` in it. Then run `sd_jwt` specifying the custom settings path:
+ - Follow the instructions in https://github.com/martinthomson/i-d-template/blob/main/doc/SETUP.md to setup the build environment.
+ - Install the sd-jwt library as described in https://github.com/danielfett/sd-jwt\
```
Comment (bc-pi) [inline README.md:28]:
```suggestion
 - Install the sd-jwt library as described in https://github.com/danielfett/sd-jwt
```

**Review by bc-pi — APPROVED** [review]

**Comment by danielfett** [comment]
I actually meant to have a line break before and after the "or", but I guess the (here intended) trailing white space was eaten by my VS Code (because I configured it to do so, duh!). I'm fine with the current version.

**Comment by bc-pi** [comment]
I was just trying to fix the 404 from https://github.com/danielfett/sd-jwt\

**Review by Sakurann — APPROVED** [review]

**Pull request merged**